dev/ProgressRing/ProgressRing.cpp

```diff
@@ -132,10 +210,13 @@ void ProgressRing::SetLottieForegroundColor(winrt::impl::com_ref<AnimatedVisuals
         }
     }();
 
-    progressRingIndeterminate->GetThemeProperties(compositor).InsertVector4(s_ForegroundName, SharedHelpers::RgbaColor(foregroundColor));
+    if (const auto progressRing = animatedVisualSource.try_as<AnimatedVisuals::ProgressRingIndeterminate>())
```

Comment:
>ProgressRingIndeterminate [](start = 79, length = 25)

does this work with determinate rings?

Reply:
Yes, for some reason. Perhaps we can sync up on this?

Reply:
did you figure this out?

---
In reply to: [434287553](https://github.com/microsoft/microsoft-ui-xaml/pull/2584#discussion_r434287553) [](ancestors = 434287553)